app/src/main/java/doit/study/droid/topic/TopicView.kt

```diff
@@ -0,0 +1,9 @@
+package doit.study.droid.topic
+
+data class TopicView (
```

Comment:
It is a data class, not a view

Reply:
Yes, naming is wrong, but if that data class is tightly coupled with View can I indicate it somehow?